boilerplate.js

```diff
@@ -3,12 +3,31 @@ const fs    = require('fs')
     , after = require('after')
     , cpr   = require('cpr')
 
+function adjustForLang(orig, lang, cb) {
+  if (!lang) {
+    process.nextTick(function() { cb(orig); });
+    return;
+  }
+
+  var extName = path.extname(orig),
```

Comment:
style nit: comma-first, line up variable names and `=` in the same block, no semi-colons pls